app/mailers/rrd_completed_mailer.rb

```diff
@@ -7,6 +7,8 @@ def build(submission)
     @rrd_status = submission.rrd_status
     @error_message = submission.form.dig('rrd_metadata', 'error') if @rrd_status == 'error'
     @bp_readings_count = submission.form.dig('rrd_metadata', 'med_stats', 'bp_readings_count') || 'N/A'
+    @active_medications = submission.form.dig('rrd_metadata', 'med_stats', 'medications_count') || 'N/A'
```

Comment:
I would be a fan of including `_count` in the naming here for consistency (non-blocking)